packages/gatsby/src/redux/actions/internal.ts

```diff
@@ -117,6 +119,23 @@ export const queryExtracted = (
   }
 }
 
+/**
+ * Set Definitions for fragment extraction, etc.
+ *
+ * Used by developer tools such as vscode-graphql & graphiql
+ *
+ * query-compiler.js.
+ * @private
+ */
+export const setDefinitions = (
+  definitionsByName: Map<string, IDefinitionMeta>
+): ISetDefinitionsAction => {
+  return {
+    type: `SET_DEFINITIONS`,
+    payload: definitionsByName,
```

Comment:
I think `SET_DEFINITIONS` is too broad.

Can we rename it to `SET_GRAPHQL_DEFINITIONS` (and also change `setDefinitions` to `setGraphQLDefinitions`, `IDefinitionMeta` to `IGraphQLDefinitionMeta`, `ISetDefinitionsAction` to `ISetGraphQLDefinitionsAction`).

Reply:
should we also change `SET_SCHEMA` to `SET_GRAPHQL_SCHEMA`?

Reply:
Sadly, this will be a subtle breaking change. There are plugins hooking to this action even if it is not a part of the public API. So let's keep `SET_SCHEMA` as is.

Reply:
sounds good!